app/views/recipes/index.html.erb

```diff
@@ -1,2 +1,21 @@
+<h1><%= @user.name.capitalize %>'s Recipes</h1>
+
+<div class="add-recipe-container">
+  <%= link_to new_recipe_path, class: "" do %>
+    <%= content_tag :i, nil, class: "" %>
```

Comment:
Kindly I suggest that if you won't add any classes to the link_to tag you might as well remove the class attribute, as shown:

```
<%= link_to new_recipe_path do %>
    <%= content_tag :i, nil %>
```